TLM/Benchmarks/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Benchmarks")]
```

Comment:
Could use some reasonable descriptions here, the project name, team name, etc. Trademark: free software whatever license we use.

Reply:
Is the SkyTools stuff from Real Time mod by dymanoid?

Reply:
Not really from Real Time, but standalone package created by dymanoid and published to `NuGet`